build.ps1

```diff
@@ -33,7 +33,7 @@ cd $PSScriptRoot
 $repoFolder = $PSScriptRoot
 $env:REPO_FOLDER = $repoFolder
 
-$koreBuildZip="https://github.com/aspnet/KoreBuild/archive/dev.zip"
+$koreBuildZip="https://github.com/aspnet/KoreBuild/archive/stable-cache.zip"
```

Comment:
I think this is going to cause issues in universe builds. The CLI and shared runtime are locked by the outer msbuild process, so this will fail. What does it take to not do this?